npm/config/config.go

```diff
@@ -17,7 +17,7 @@ var DefaultConfig = Config{
 		EnablePrometheusMetrics: true,
 		EnablePprof:             true,
```

Comment:
Just check-in:
Based on our discussion, it would be better to set `EnablePprof` as `false`?

Reply:
Oops. Reverted! nevermind.